addons/io_sketchfab_plugin/blender/imp/gltf2_blender_material.py

```diff
@@ -81,6 +81,14 @@ def blender_alpha(gltf, material_idx):
         material = bpy.data.materials[pymaterial.blender_material]
 
         node_tree = material.node_tree
+
+        #Fix alphas for 2.8
+        if bpy.app.version == (2, 80, 0):
```

Comment:
Is there any reason for `==` instead of  `>=` used in other places?

Reply:
Yep, not looking for them.
Fixed in the latest force-push.